# 自定义validator
from rest_framework.exceptions import ValidationError